src/extension/background-script/router.ts

```diff
@@ -83,6 +84,7 @@ const routes = {
   // Public calls that are accessible from the inpage script (through the content script)
   public: {
     webbtc: {
+      providerOnboard: onboard.providerOnboard,
```

Comment:
the naming feels wrong here. It should be a verb. "setupProvider"? does it do more than "setupMasterKey"?

Reply:
let's use openOnboardingPrompt as it can be used for other purposes or other upcoming providers as well, not just for setupMasterKey

Reply:
Sounds good, thanks!